docs/customizing.md

```diff
@@ -180,6 +183,58 @@ Example `pulp_install_plugins`:
          upgrade: true
 ```
 
+### Latest patch release for a minor release of pulpcore
+
+Initial installation:
+
+1. Make sure you are running the latest version of the installer, which installs pulpcore's
+   latest 3.y release.
+1. Confirm that all the latest stable releases of your desired plugins are compatible with
+   pulpcore's latest version , such as by reading the release announcement email thread for
+   pulpcore's latest version, reading the plugins README, or as a last resort, reading their `setup.py`.
+1. Set `pulp_install_plugins` with each plugin listed as a key, with an empty enum '{}' as each plugin's value.
+1. Run `pulp_installer`.
+1. Make sure to save your variables/playbook for later usage.
+
+Example `pulp_install_plugins`:
+```
+  vars:
+    pulp_install_plugins:
+      pulp-container: {}
+      pulp-file: {}
+      pulp-rpm: {}
```

Comment:
How about they specify the branch during initial install?

So that they can more easily update in the future. Updating in the future would just involve adding the "update: true".